src/services/chris_integration.tsx

```diff
@@ -212,6 +212,18 @@ class ChrisIntegration {
     return (await client.getPACSFiles({ fname_exact: imgTitle })).data
   }
 
+  static async getFilePathNameByUID(StudyInstanceUID: string, SeriesInstanceUID: string): Promise<string> {
+    let client: any = await ChrisAPIClient.getClient();
+    
+    const res = await client.getPACSFiles({
+      StudyInstanceUID,
+      SeriesInstanceUID,
+      limit: 1
```

Comment:
If you are using `limit`, are we planning to paginate the response? In that case, I suggest adding `page: number, perPage: number` parameters to the function signature as well as add `offset` to the request parameters.